### PR TITLE
added context-click, modified eclipse-formatter

### PR DESCRIPTION
**Pull request by dbe-it**
Added simulated context-click and integration-test;
modified eclipse-formatter:
-no indent javadoc-tags
-no whitespace before&after opening parenthesis in annotations
-no whitespace before closing parenthesis in annotations
-no whitespace after constructor- and methods-name

### CODE REVIEW

**Comment by slu-it** [comment]
@dbe-it sollte 1.1.0 sein, da das die nächste version auf diesem branch ist...

**Comment by slu-it** [comment]
also missing documentation on:
documentation/chapters/mouse.md

File: development/eclipse_formatter.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
 <setting id="org.eclipse.jdt.core.formatter.insert_space_after_comma_in_multiple_local_declarations" value="insert"/>
 <setting id="org.eclipse.jdt.core.formatter.alignment_for_arguments_in_enum_constant" value="48"/>
 <setting id="org.eclipse.jdt.core.formatter.insert_space_after_opening_angle_bracket_in_parameterized_type_reference" value="do not insert"/>
-<setting id="org.eclipse.jdt.core.formatter.comment.indent_root_tags" value="true"/>
+<setting id="org.eclipse.jdt.core.formatter.comment.indent_root_tags" value="false"/>
 <setting id="org.eclipse.jdt.core.formatter.wrap_before_or_operator_multicatch" value="true"/>
```
Comment (benhamidene) [inline development/eclipse_formatter.xml]: changing the formatter means updating all files .. is that really necessary?

Comment (slu-it) [inline development/eclipse_formatter.xml]: yes, we changed the formatter when we published to git hub and reformatted all files. But something went wrong with the eclipse settings and some of the format settings were not set correctly. This fixes the formatter rules to conform to the actual code

File: webtester-core/src/test/java/integration/utils/MouseIntegrationTest.java [hunk 2]
```diff
@@ -39,6 +38,19 @@ public void testThatMovingTheMouseInSequenceIsPossible() {
     }
 
     @Test
+    public void testThatContextClickingAPageObjectWithTheMouseIsPossible() {
+
+        MoveTestPage page = getBrowser().create(MoveTestPage.class);
+
+        /* Context click the context click button. */
+        Mouse.contextClick(page.clickContextClickButton);
```
Comment (benhamidene) [inline webtester-core/src/test/java/integration/utils/MouseIntegrationTest.java]: I would use a getter Methid instead...

**Pull request merged**